io.github.dosbox-staging.yml

```diff
@@ -0,0 +1,52 @@
+app-id: io.github.dosbox-staging
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+
+command: dosbox
+
+build_options:
+  cppflags: -DNDEBUG
+  cflags: -O3
+  cxxflags: -O3
+
+finish-args:
+  - --device=all # we need OpenGL and controller access
+  - --share=network # IPX and serial port emulation over UDP
+  - --socket=x11 # default to X11, as Gnome does not provide SSD
```

Comment:
If it works natively on Wayland, this line should be changed from `x11` to `fallback-x11`.

Reply:
It works natively on Wayland, but it's really poor integration - it's still quite buggy and completely unworkable on Gnome (because Gnome does not support server-side-decoration extensions).

To limit the amount of pain users will experience, I would prefer to keep it working via X11/XWayland by default, but allow users to opt-in to Wayland if they wish. I am not sure if options I selected here allow this.

Reply:
`--socket=wayland` should be dropped then. Users who want to test Wayland support will need to use Flatseal or CLI to enable Wayland socket.

Reply:
@barthalion Thank you! Done :)